slob/README.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Serialized LOB
+category: Structural
+language: en
+tag:
+  - Data access
+---
+
+## Intent
+
+* Object models often contain complicated graphs of small objects. Much of the information in these
+  structures isn’t in the objects but in the links between them.
+* Objects don’t have to be persisted as table rows related to each other.
+* Another form of persistence is serialization, where a whole graph of objects is written out as a
+  single large object (LOB) in a table.
+
+## Explanation
+
+**In plain words**
+
+> The Forest here represents the object graph.
+> A forest contains animals and plants. As is in real life the forest object contains plants and
+> animals where some animals eat other animals and some eat only plants.
+> * These relationships are maintained in the Forest Object.
+> * There are 2 types of Serializers available ClobSerializer and BlobSerializer.
+> * ClobSerializer uses character or textual based serialization, here the Object graph is converted
+> * to XML and then stored as text in DB.
+> * BlobSerializer uses binary data for serialization, here the Object graph is converted to Byte
+> * Array and then stored as a blob in DB.
+
+**Programmatic Example**
+
+* Here is the `Animal` class. It represents the Animal object in the Forest. It contains the name of
+  the animals in the forest and details of what they eat from the forest plants/animals or both.
+
+```java
+import static com.iluwatar.slob.lob.Animal.iterateXmlForAnimalAndPlants;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
```

Comment:
The code snippets included in the `README.md` should be minimal. We don't want to include irrelevant parts of the code, just the things needed to explain the pattern. See the other patterns for examples of how to do this.

Reply:
Got it, removed import statmenets and license doc from README.md. Kindly check now